src/_includes/sidenav-level-2.html

```diff
@@ -20,7 +20,9 @@
     >{{entry2.title}}
     {%- if entry2.permalink == nil %} (TBC)
     {%- elsif entry2.permalink contains '://' -%}
-      <i class="fas fa-external-link-alt"></i>
+      <span class="material-icons md-24">
+        description
```

Comment:
I haven't had a chance to take a closer look at these changes yet, but why `description` here? I may be mistaken but I think the external icon we use elsewhere is called `launch`: https://fonts.google.com/icons?icon.query=launch